azext_iot/deviceupdate/_help.py

```diff
@@ -0,0 +1,252 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for Device Update commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_deviceupdate_help():
+
+    helps["iot device-update"] = """
+        type: group
+        short-summary: Device Update for IoT Hub is a service that enables you to deploy over-the-air updates (OTA) for your IoT devices.
+        long-summary: As organizations look to further enable productivity and operational efficiency, Internet of Things (IoT) solutions
+          continue to be adopted at increasing rates. This makes it essential that the devices forming these solutions are built on a foundation
+          of reliability and security and are easy to connect and manage at scale. Device Update for IoT Hub is an end-to-end platform that customers
+          can use to publish, distribute, and manage over-the-air updates for everything from tiny sensors to gateway-level devices.
+    """
+
+    helps["iot device-update account"] = """
+        type: group
+        short-summary: Device Update account management.
+    """
+
+    helps["iot device-update account create"] = """
+        type: command
+        short-summary: Create a Device Update account.
+        long-summary: This command may also be used to update an existing account provided the generic update options
+          in the update command are not needed.
+
+        examples:
+        - name: Create a Device Update account in target resource group using the resource group location.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group}
+
+        - name: Create a free sku Device Update account in target resource group with specified location and tags without polling.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group} -l westus --tags a=b c=d --sku Free --no-wait
+
+        - name: Create a Device Update account in target resource group with a system managed identity.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group} --assign-identity [system]
+
+        - name: Create a Device Update account in target resource group with a system managed identity then
+                assign the system identity to one or more scopes (space-separated) with the role of Contributor.
```

Comment:
do you need to say one or more scopes when the example only has one scope? Maybe add a second scope here and make the next help only have one scope?